src/wp-includes/functions.php

```diff
@@ -6201,6 +6201,8 @@ function validate_file( $file, $allowed_files = array() ) {
 
 	// Normalize path for Windows servers
 	$file = wp_normalize_path( $file );
+	// Normalize path for $allowed_files as well so it's an apples to apples comparison
+	$allowed_files = array_map( 'wp_normalize_path', $allowed_files );
```

Comment:
Minor coding standard nit:

Full stop inline comments.

```suggestion
	// Normalize path for Windows servers.
	$file = wp_normalize_path( $file );
	// Normalize path for $allowed_files as well so it's an apples to apples comparison.
	$allowed_files = array_map( 'wp_normalize_path', $allowed_files );
```